clients/client-web/package.json

```diff
@@ -4,7 +4,7 @@
   "main": "build/index.js",
   "author": "Eli Perelman <eli@eliperelman.com>",
   "license": "MPL-2.0",
-  "repository": "taskcluster/taskcluster-client-web",
+  "repository": "taskcluster/taskcluster",
```

Comment:
Thoughts on https://github.com/taskcluster/taskcluster/tree/master/clients/client-web instead?

Reply:
Huh, judging from https://yarnpkg.com/en/docs/package-json#toc-repository even this syntax is invalid, but yarn figured it out.  I'll make the suggested change (and to tc-client)